src/univariates.jl

```diff
@@ -78,26 +78,19 @@ Get the degrees of freedom.
 dof(d::UnivariateDistribution)
 
 """
-    minimum(d::UnivariateDistribution)
+    minimum(d::Distribution)
 
 Return the minimum of the support of `d`.
 """
 minimum(d::UnivariateDistribution)
 
 """
-    maximum(d::UnivariateDistribution)
+    maximum(d::Distribution)
 
 Return the maximum of the support of `d`.
 """
 maximum(d::UnivariateDistribution)
```

Comment:
Can you move these lines to `src/common.jl` and replace `d::UnivariateDistribution` with `d::Distribution`?